charmcraft/commands/build.py

```diff
@@ -412,10 +437,22 @@ def process(self, parsed_args):
         """Process the received options."""
         result = {}
         for opt in self._options:
-            meth = getattr(self, "validate_" + opt)
+            print(parsed_args)
```

Comment:
leftover print

Reply:
Whoops, accidentally included this bit. Removed